common.py

```diff
@@ -541,6 +542,13 @@ def git_repo_setup(repo_dir: str, *, repo_wipe: bool = True, url: str, branch: s
     Repo.clone_from(url, repo_dir, branch=branch)
 
 
+def github_repo_setup(repo_dir: str, *, repo_wipe: bool = True, url: str, pullnumber: str) -> None:
```

Comment:
Maybe this function name needs work, it is not obvious at a glace the difference between this function and "git_repo_setup"

Reply:
Also, not sure this belongs in common if we are hardcoding pulling from the dpu-operator repo. No one else will use this but dpu operator post-config as it is currently written.

Reply:
Yes I agree I will move it out of common! I don't think the function name is an issue because git repo setup set's up a git repository without reading from the gihtub api whereas github repo setup does. IMO that explains the difference.

Reply:
To me this function is doing "get_dpu_operator_pr(pollnumber: int)"

Or even something like
```
repo, branch = get_repo_from_poll_num(poll_number) -> (repo: str, branch: str)
git_repo_setup(repo_dir, repo_wipe=true, url=repo, branch=branch)
```

Reply:
okay I will rename it "clone_dpu_operator_pr(pollnumber: int)"

Reply:
removed this function (building from branch)